Libraries/PlayerSettings.php

```diff
@@ -296,17 +296,17 @@ function GetSettingsUI($player)
   $rv .= CreateRadioButton($SET_Cardback . "-" . 11, "Default", 26, $SET_Cardback . "-" . $settings[$SET_Cardback], "Padawan Unlimited");
   $rv .= CreateRadioButton($SET_Cardback . "-" . 12, "Default", 26, $SET_Cardback . "-" . $settings[$SET_Cardback], "RVA SWU");
 
-  foreach(PatreonCampaign::cases() as $campaign) {
-    if(isset($_SESSION[$campaign->SessionID()]) || (isset($_SESSION["useruid"]) && $campaign->IsTeamMember($_SESSION["useruid"]))) {
-      $hasCardBacks = true;
-      $cardBacks = $campaign->CardBacks();
-      $cardBacks = explode(",", $cardBacks);
-      for($i = 0; $i < count($cardBacks); ++$i) {
-        $name = $campaign->CampaignName() . (count($cardBacks) > 1 ? " " . $i + 1 : "");
-        $rv .= CreateRadioButton($SET_Cardback . "-" . $cardBacks[$i], str_replace(' ', '', $name), 26, $SET_Cardback . "-" . $settings[$SET_Cardback], $name);
-      }
-    }
-  }
+  // foreach(PatreonCampaign::cases() as $campaign) {
```

Comment:
were these meant to be commented out?

Reply:
These comments were not intended to be included - they are used to enable Undo functionality during local development. Thank you for letting me know.